### ISSUE TITLE
AppImage error on Fedora 37

### ISSUE THREAD

**Issue by ionutrogojan** [rustdesk/rustdesk#1691]
**Describe the bug you encountered:**

The Appimage will not open after getting executable permissions
```sh
/tmp/.mount_RustDekDqfIn//usr/bin/rustdesk: error while loading shared libraries: libnsl.so.1: cannot open shared object file: No such file or directory
```
**What did you expect to happen instead?**

I expected the program to open

**How did you install `RustDesk`?**
```
chmod a+x RustDesk-1.1.10-x86_64_1804.AppImage
./RustDesk-1.1.10-x86_64_1804.AppImage
```

**RustDesk version and environment**

`RustDesk-1.1.10-x86_64_1804.AppImage`
```
OS: Fedora Linux 37 (Workstation Edition Prerelease) x86_64 
Kernel: 5.19.14-300.fc37.x86_64 
Shell: bash 5.2.2 
Resolution: 1920x1080 
DE: GNOME 43.0 
WM: Mutter 
CPU: AMD Ryzen 5 5600X (12) @ 3.700GHz 
GPU: AMD ATI Radeon RX 6700/6700 XT/6750 XT / 6800M 
Memory: 3474MiB / 32008MiB
```

**Comment by rustdesk** [comment]
appimage support is removed.

**Comment by rustdesk** [comment]
https://twitter.com/rustdesk

**Comment by cooperbang** [comment]
> it is back. https://twitter.com/rustdesk

In Fedora 37 (using Wayland) it only works if you start the Appimage in XWayland.
--> GDK_BACKEND=x11 ./rustdesk-1.2.0-x86_64.AppImage

When trying to start in Wayland it fails with Error
(rustdesk:8523): GLib-GIO-ERROR **: 16:23:28.318: Settings schema 'org.gnome.settings-daemon.plugins.xsettings' does not contain a key named 'antialiasing'

**Comment by rustdesk** [comment]
https://github.com/Ultimaker/Cura/issues/12776
https://github.com/powenn/AltServer-LinuxGUI/issues/3

**Comment by rustdesk** [comment]
> GDK_BACKEND=x11

@cooperbang Thank you very much for pointing out this. We will add it into desktop.ini

**Comment by cooperbang** [comment]
I think the AppImage sources should be updated to Ubuntu Jammy Jellyfish (22.04.1).

```
 sources:
    - sourceline: deb http://archive.ubuntu.com/ubuntu/ jammy main restricted
    - sourceline: deb http://archive.ubuntu.com/ubuntu/ jammy-updates main restricted
    - sourceline: deb http://archive.ubuntu.com/ubuntu/ jammy universe
    - sourceline: deb http://archive.ubuntu.com/ubuntu/ jammy-updates universe
    - sourceline: deb http://archive.ubuntu.com/ubuntu/ jammy multiverse
    - sourceline: deb http://archive.ubuntu.com/ubuntu/ jammy-updates multiverse
    - sourceline: deb http://archive.ubuntu.com/ubuntu/ jammy-backports main restricted universe multiverse
    - sourceline: deb http://ppa.launchpad.net/pipewire-debian/pipewire-upstream/ubuntu jammy main
```

I´m not sure - but it seems like it´s fixed in Ubuntu 22.04.1.
--> https://bugs.launchpad.net/ubuntu/+source/ubuntu-gnome-default-settings/+bug/1950590

**Comment by rustdesk** [comment]
> 22.04.1

@Kingtous
